### PR TITLE
DIV-8653 | adding swagger api docs

### PR DESCRIPTION
**Pull request by saiprakash-v**
adding swagger api docs in vc-certification and vc-management services
@egov-joy @srprasanna @varadeth Please review

### CODE REVIEW

File: backend/vc-certification-service/certification-service-swagger.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+swagger: '2.0'
```
Comment (egov-joy) [inline backend/vc-certification-service/certification-service-swagger.yml:1]: can we use version 3 of the spec instead?

Comment (prasanna-egov) [inline backend/vc-certification-service/certification-service-swagger.yml:1]: Lets use OpenAPI 3.0 specification.

File: backend/vc-management-service/management-service-swagger.yml [hunk 2]
```diff
@@ -0,0 +1,155 @@
+swagger: '2.0'
```
Comment (prasanna-egov) [inline backend/vc-management-service/management-service-swagger.yml:1]: OpenAPI 3.0 spec here as well.

**Review by prasanna-egov — APPROVED** [review]
Approved.

**Pull request merged**